test/Microsoft.AspNetCore.Server.Kestrel.Performance/project.json

```diff
@@ -8,7 +8,7 @@
     "netcoreapp1.0": {
       "dependencies": {
         "Microsoft.NETCore.App": {
-          "version": "1.0.1-*",
```

Comment:
Wasn't sure it would work; BenchMarkDotnet currently doesn't like `netcoreapp1.1`